src/plugins/data/public/ui/dataview_picker/change_dataview.tsx

```diff
@@ -0,0 +1,221 @@
+    >
+      <EuiPopover
+        panelClassName="changeDataViewPopover"
+        button={createTrigger()}
+        panelProps={{
+          ['data-test-subj']: 'changeDataViewPopover',
+        }}
+        isOpen={isPopoverOpen}
+        closePopover={() => setPopoverIsOpen(false)}
+        panelPaddingSize="none"
+        initialFocus={`#${searchListInputId}`}
+        buffer={8}
+      >
+        <div style={{ width: POPOVER_CONTENT_WIDTH }}>
+          {onAddField && (
+            <>
+              <EuiContextMenuPanel
+                size="s"
+                items={[
+                  <EuiContextMenuItem
+                    key="add"
+                    icon="indexOpen"
+                    data-test-subj="indexPattern-add-field"
+                    onClick={() => {
+                      setPopoverIsOpen(false);
+                      onAddField();
+                    }}
+                  >
+                    {i18n.translate('data.query.queryBar.indexPattern.addFieldButton', {
+                      defaultMessage: 'Add a field to this data view...',
+                    })}
+                  </EuiContextMenuItem>,
+                  <EuiContextMenuItem
+                    key="manage"
+                    icon="indexSettings"
+                    data-test-subj="indexPattern-manage-field"
+                    onClick={() => {
+                      setPopoverIsOpen(false);
+                      application.navigateToApp('management', {
+                        path: `/kibana/indexPatterns/patterns/${currentDataViewId}`,
+                      });
+                    }}
+                  >
+                    {i18n.translate('data.query.queryBar.indexPattern.manageFieldButton', {
+                      defaultMessage: 'Manage this data view...',
+                    })}
+                  </EuiContextMenuItem>,
+                ]}
+              />
+              <EuiHorizontalRule margin="none" />
+            </>
+          )}
+          <DataViewsList
+            dataViewsList={dataViewsList}
+            onChangeDataView={(newId) => {
+              onChangeDataView(newId);
+              setPopoverIsOpen(false);
+            }}
+            currentDataViewId={currentDataViewId}
+            selectableProps={selectableProps}
+            searchListInputId={searchListInputId}
+          />
+          {onDataViewCreated && (
+            <>
+              <EuiHorizontalRule margin="none" />
+              <EuiContextMenuItem
+                css={css`
+                  color: ${euiTheme.colors.primaryText};
+                `}
+                data-test-subj="dataview-create-new"
+                icon="plusInCircleFilled"
+                onClick={() => {
+                  setPopoverIsOpen(false);
+                  onDataViewCreated();
+                }}
+              >
+                {i18n.translate('data.query.queryBar.indexPattern.addNewDataView', {
+                  defaultMessage: 'New data view...',
```

Comment:
```suggestion
                  defaultMessage: 'New data view',
```

Reply:
Should this be "Create a data view"?

Reply:
Yep, sorry bad copy from mock, should have stuck with existing text.